notebooks/Emukit-Bayesian-optimization-benchmark.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Emukit Bayesian optimization Benchmark\n",
+    "\n",
+    "This notebook compares multiple Bayesian optimization methods against each other, using the Branin test function."
```

Comment:
multiple -> two. I know "two" is a subset of "multiple" but it sounds a bit exaggerated somehow.